scripts/qa_crowbarsetup.sh

```diff
@@ -1217,12 +1219,20 @@ function onadmin_crowbar_register()
 
     local adminfqdn=`crowbar machines list | grep crowbar`
     local adminip=`knife node show $adminfqdn -a crowbar.network.admin.address | awk '{print $2}'`
+
+    if [[ $keep_existing_hostname = 1 ]] ; then
```

Comment:
I am fine with the syntax as well. I even thought about a bigger cleanup commit to switch consistently to double brackets: "[[ ... ]]".
Btw. we also have variables that officially support to set them to 0, because it means to disable it even if the default would be to enable it (compare $want_ceph and $want_swift).

:+1: